roles/insert_dns_records/defaults/main.yml

```diff
@@ -21,3 +21,6 @@ use_dhcp: false
 dhcp_lease_time: 24h
```

Comment:
dhcp_lease_time should be reduced to avoid lease kruft.

Reply:
I'll make a new PR for that :)